site/layouts/partials/topic.html

```diff
@@ -12,7 +12,7 @@
   </a>
   {{ end }}
   <span class="topic-text-wrapper">
-    <a class="link-unstyled text-dark font-weight-bold coming-soon" href="#">
+    <a class="link-unstyled text-dark font-weight-bold course-info-topic" href="#">
```

Comment:
Should the `coming-soon` style be added back for these?

Reply:
yeah, that's a good idea